gradoop-temporal/src/main/java/org/gradoop/temporal/util/TimeFormatConversion.java

```diff
@@ -35,4 +36,15 @@ public static long toEpochMilli(LocalDateTime time) {
     Objects.requireNonNull(time);
     return time.toInstant(ZoneOffset.UTC).toEpochMilli();
   }
+
+  /**
+   * Converts milliseconds since Unix epoch to the respective {@link LocalDateTime} object.
```

Comment:
This sounds weird to me, maybe just
```suggestion
   * Converts a unix timestamp as a {@code long} to a {@link LocalDateTime} object.
```